### ISSUE TITLE
Adding multiple annotation bodies at the same time

### ISSUE THREAD

**Issue by Apeli** [recogito/recogito-js#39]
Hi, I'm trying to make a list of preselected values that would add a comment + colorcode to the annotation at the same time. I took the ColorSelectorWidget from https://recogito.github.io/guides/extending-the-editor/ as the base. I got to the point where I have the buttons, but when clicking on one, I can only set one body per click. So calling addTag the actual annotation only has 1 body, which ever is called last. 

I tried like this:
```
var addTag = function(evt) {
args.onAppendBody({
                type: 'TextualBody',
                purpose: 'highlighting',
                value: evt.target.dataset.color,
            });

            args.onAppendBody({
                type: 'TextualBody',
                purpose: 'commenting',
                value: evt.target.dataset.tag,
            });
            ...
```
There the annotation only has the 'commenting' -body. Is there a way to set multiple bodies at the same time?

**Comment by rsimon** [comment]
Good point - yes, interestingly no-one raised that as a requirement so far. It's not currently possible to add multiple bodies in one go. The onAppendBody method is async. Therefore, the second call essentially overwrites the first one. 

If this is urgent: an ugly hack would be to delay the second call by a few milliseconds.

```js
setTimeout(() => args.appendBody({...}), 1); // 1 might be sufficient to put it at the end of the queue
``` 

But for a clean solution, I'd need to extend the API with `.onAppend/Update/UpsertBodies()` methods. (Or simply make the current methods accept a single body or an array.)
